wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/domain/ereignis/EreignisartEntity.java

```diff
@@ -0,0 +1,22 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.domain.ereignis;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.rest.ereignis.EreignisartDTO;
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.service.EreignisartModel;
+
+public enum EreignisartEntity {
```

Comment:
`Entity` bitte am Ende entfernen. Entitäten und deren zusätzlich verwendete Klassen haben aktuell keinen Suffix.

Reply:
siehe commit [b1a086e](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/b1a086e3245ecd720cc3083f9e237c025716cf4d)